Wizardry/Common/ItemSys/Source/StaffHeal.c

```diff
@@ -73,6 +73,15 @@ void ExecStandardHeal(ProcPtr proc)
     amount = HealAmountGetter(amount, unit_act, unit_tar);
 #endif
 
+
+#if defined(SID_LiveToServe) && (COMMON_SKILL_VALID(SID_LiveToServe))
+    if (SkillTester(unit_act, SID_LiveToServe))
+    {
+        AddUnitHp(unit_act, -(gBattleTarget.unit.curHP - GetUnitCurrentHp(unit_tar)));
```

Comment:
apparently the calculation is not consistent to the skill desc

Reply:
True, it's missing the skill holder's MAG value. Good spot. Not sure how I missed that.